src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,49 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+import ladder.domain.dto.LineResponseDto;
+import ladder.domain.randomGenerator.RungGenerator;
+
+public class Line {
+
+    private static final boolean FIRST_POSITION_NOT_EXIST = false;
+    private static final int EXCLUDE_LAST_POSITION = 1;
+    private static final int FOR_BEFORE_POSITION = 1;
+
+    private final RungGenerator rungGenerator;
+    private final List<Rung> rungs;
+
+    public Line(final RungGenerator rungGenerator, final int personCount) {
+        this.rungGenerator = rungGenerator;
+        this.rungs = makeRungs(personCount);
+    }
+
+    private List<Rung> makeRungs(int personCount) {
+        List<Rung> rungs = new ArrayList<>();
+
+        IntStream.range(0, personCount - EXCLUDE_LAST_POSITION)
+                .forEach(currentPosition -> rungs.add(generateRung(currentPosition, rungs)));
+
+        return rungs;
+    }
+
+    private Rung generateRung(int currentPosition, List<Rung> rungs) {
+        if (!hasRungDuplicated(currentPosition, rungs) && rungGenerator.getRandomBooleanStatus()) {
```

Comment:
부정연산자는 가독성을 떨어뜨린다는 의견이 있어서, 피할 수 있으면 피하는게 좋습니다.
hasNotRungDuplicated (네이밍이 구린데요.. 아무튼 중복안됨의 의미를 담는 것으로 새로 지어주셔요..) 로 변경해주시거나 하나 새로 뚫어주세요. 
```suggestion
    if (hasNotRungDuplicated(currentPosition, rungs) && rungGenerator.getRandomBooleanStatus()) {
```
저라면 리팩토링하기 귀찮으니(헷갈리고 머리아프니) 다음처럼 하나 더 뚫을거 같아요

```java
private boolean hasNotRungDuplicated(int currentPosition, List<Rung> rungs) {
   return !hasRungDuplicated(currentPosition, rungs)
}
```

아주 중요한 사항은 아니고 권장에 가까우니 반영 안 하셔도 됩니다

Reply:
경험이 풍부한 웨지와 저같은 뉴비의 차이가 이런부분 부터에서 시작 아닐까요 ㅎㅎㅎ
이런 미세 꿀팁 너무 좋아합니다. 바로 반영하겠습니다 😄